<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measuring_Weight</class>
 <widget class="QWidget" name="Measuring_Weight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnMeasure">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>再次测量</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.clipboard.variant.edit.png</normaloff>:/imgIcon/appbar.clipboard.variant.edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>400</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>保存体重</string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.disk.upload.png</normaloff>:/imgIcon/appbar.disk.upload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>520</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
	border-radius:5px;
}

QPushButton:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string> 退出    </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.2.png</normaloff>:/imgIcon/appbar.close.2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: rgb(50 , 204 , 194);
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>测量体重</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblWeight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>92</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: rgb(50 , 204 , 194);
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>500.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTipsIcon">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>421</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imgIcon/mesure.png);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTips">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	
	color: rgb(255, 255, 255);
	background-color:rgb(50 , 204 , 194);
}</string>
   </property>
   <property name="text">
    <string>请站到体重秤上</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtBserCom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>521</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(50 , 204 , 194);
background-color: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxComment">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>561</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="title">
    <string>评语：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblKg">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>92</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: rgb(50 , 204 , 194);
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDetail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>701</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: rgb(50 , 204 , 194);
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>BMI指数&lt;15.8&gt;, 体态&lt;极度偏瘦&gt;, 时间&lt;2017-03-01 12:30:25&gt;</string>
   </property>
  </widget>
  <zorder>groupBoxComment</zorder>
  <zorder>btnMeasure</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnClose</zorder>
  <zorder>lblInfo</zorder>
  <zorder>lblWeight</zorder>
  <zorder>lblTipsIcon</zorder>
  <zorder>lblTips</zorder>
  <zorder>txtBserCom</zorder>
  <zorder>lblKg</zorder>
  <zorder>lblDetail</zorder>
 </widget>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
